common.proto

```diff
@@ -70,7 +72,8 @@ message MentionEvent {
 }
 
 // CommandEvent will be sent when a user issues a command in a channel. Commands
-// cannot be issued via private message.
+// cannot be issued via private message. If a message is parsed as a command, it
+// SHOULD NOT be sent as another message type.
```

Comment:
I'm not a huge fan of "SHOULD NOT". It doesn't give the client any guarantees. Instead, I'd say:

```
If a message is parsed as a command, plugins must not expect to receive the message as another type as well.
```

Reply:
I'm actually going to change this to a MUST NOT to make the chat backend contract clearer